R/model.R

```diff
@@ -45,7 +45,8 @@ model_compare <- function(state, observed, pars) {
   }
 
   if (pars$use_new_compare == 1) {
-    ll_cases <- ll_nbinom(Y, newI, kappa = pars$kappa_cases, pars$exp_noise)
+    cases <- ceiling(newI * delta)
+    ll_cases <- ll_nbinom(Y, cases, kappa = pars$kappa_cases, pars$exp_noise)
```

Comment:
```suggestion
    ll_cases <- ll_nbinom(Y, cases, pars$kappa_cases, pars$exp_noise)
```

Just tidying this up as we should not use named args before unnamed ones